assets/js/blocks/cart-checkout/cart/full-cart/utils.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { decodeEntities } from '@wordpress/html-entities';
+import { PLACEHOLDER_IMG_SRC } from '@woocommerce/block-settings';
+
+/**
+ * Return a currency value as a number for doing calculations.
+ * Note this doesn't convert into dollars, currency values are in minor units (e.g. cents).
+ *
+ * @param {string} currencyValue Currency value string (in minor unit).
+ * @return {number} The currency value as int (in minor unit).
+ */
+export const getPriceNumber = ( currencyValue ) => {
+	return parseInt( currencyValue, 10 );
```

Comment:
I know that was inherited code, but I'm wondering if we really need this function. While reviewing this PR and seeing calls to `getPriceNumber()` I wasn't sure what that was doing and needed to check every time just to find out it's only calling `parseInt()`.

I think code would be clearer if we simply called `parseInt()` everywhere we are currently calling `getPriceNumber()`, at least that would reduce mental load for me.

But maybe that's just my personal taste, so feel free to take it or ignore me. :slightly_smiling_face:

Reply:
Agree, just left it there because it already existed.